docs/setup-cluster/basic-setup-guide/basic.rst

```diff
@@ -0,0 +1,56 @@
+.. _basic-setup:
+
+.. _install-cluster:
+
+#############
+ Basic Setup
+#############
+
+If you have not used Determined before, and you want to quickly set up a new training environment,
+you are at the right place!
+
+.. note::
+
+   To set up a Determined training environment on-prem or on cloud, visit :ref:`Advanced Setup
+   <advanced-setup>`.
```

Comment:
This needs re-phrasing; technically we _are_ about to set up a Determined training environment on-prem.

Maybe: 
```
To set up a Determined training environment on a shared server or cluster or in the cloud, visit :ref:`Advanced Setup <advanced-setup>`.
```

Honestly, the fact that it's called "Basic vs Advanced" is probably enough to motivate the fact that there are two docs.  This note could practically just say, "For advanced installations, see Advanced Setup".

Reply:
Edited.

For more advanced installations, visit :ref:`Advanced Setup <advanced-setup>`.